base/cvd/BUILD.crosvm.bazel

```diff
@@ -0,0 +1,41 @@
+package(
+    default_visibility = [
+        "@//:android_cuttlefish",
+    ],
+)
+
+genrule(
+    name = "wayland_virtio_gpu_metadata_impl",
+    srcs = ["gpu_display/protocol/virtio-gpu-metadata-v1.xml"],
+    outs = ["virtio-gpu-metadata-v1.c"],
+    cmd = "$(location @wayland//:wayland_scanner) -s private-code < $(location gpu_display/protocol/virtio-gpu-metadata-v1.xml) > $(location virtio-gpu-metadata-v1.c)",
+    tools = ["@wayland//:wayland_scanner"],
+)
+
+genrule(
+    name = "wayland_virtio_gpu_metadata_server_header",
+    srcs = ["gpu_display/protocol/virtio-gpu-metadata-v1.xml"],
+    outs = ["virtio-gpu-metadata-v1.h"],
+    cmd = "$(location @wayland//:wayland_scanner) -s server-header < $(location gpu_display/protocol/virtio-gpu-metadata-v1.xml) > $(location virtio-gpu-metadata-v1.h)",
```

Comment:
nit: as the rule has single file input and output we can use `$(SRCS)` and `$@` or `$(OUTS)` to avoid repeating the filenames: `$(location @wayland//:wayland_scanner) -s server-header < $(SRCS) > $(OUTS)"`

Reply:
Done!